src/use-safe-callback/index.ts

```diff
@@ -18,15 +15,16 @@ export function useSafeCallback<F extends (v: any) => any>(
 
 export function useSafeCallbackExtraDeps<
   F extends (v: any) => any,
+  T extends {[P in keyof S]: S[P] extends ExtraDeps<infer R> ? R : never},
```

Comment:
TS wasn't able to infer the type of V in ExtraDeps<V> so in lieu of the ObjMap type definition that our flowts tool generated, added this T definition to help it infer the ExtraDeps<V> type

Reply:
Woah this is fancy! I wonder if it could be valuable for you to talk in the frontend guild about how this transition worked. 

I'm not sure that we have much of this fancy `ObjMap` stuff going on in our codebase outside of this, but getting exposure to how TS works and how you migrated this type could prove useful in other spots of our code.